uppsrc/ide/SelectPkg.cpp

```diff
@@ -178,6 +178,7 @@ SelectPackageDlg::SelectPackageDlg(const char *title, bool selectvars_, bool mai
 	cancel.WhenAction = WhenClose = THISBACK(OnCancel);
 	clist.Columns(4);
 	clist.WhenEnterItem = clist.WhenKillCursor = THISBACK(ListCursor);
+	alist.AutoHideSb();
```

Comment:
I was thinking about it, however I have never decided to change that.

Reply:
I remembered why I decided to do not propose this change. It is causing flickering when you switch from assembly with relatively small amount of packages (scrollbar not needed) to the assembly that has a lot (scrollbar needed).

Reply:
I don't have flickering, i use Windows 10.
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExMTU5ZmUyZmRmNzQwMjhiMThmN2U4YWRjYTk4NTNmYTMyZTNhZmNlYSZjdD1n/7n6le3wmDmYuQWMP1b/giphy.gif)

Reply:
This is exactly what I mean - this is visually inconsistency. It looks strange when one time you have scrollbar and the other time you don't. In the end the placement of ArrayCtrl lines are different between these two views, which is not good and disrupting. So, for now I am against this change.

We need different solution to make it better. It is called overlay scrollbars. It is not yet implemented in CtrlLib, but it should give us space in many controls and one of the beneficent would be "Select main package" window.
-----------
Anyway, don't take my comments personal. I am glad that you created the PR and give the space for discussion. However, we can not accept everything.

Reply:
Yes, overlay scrollbars would be the solution.
No worries, I understand.
Thank you.